data/ignore-err-example.txt

```diff
@@ -1 +1,3 @@
 GET /api/{domain}/{project}/badges/security-score removed the success response with the status '200'
+API   removed the schema 'rules' [api-schema-removed]
```

Comment:
I think we should prefix each line by its type, like this:
```
API GET /api/{domain}/{project}/badges/security-score removed the success response with the status '200'
components removed the schema 'rules' [api-schema-removed]
...

```

Reply:
not sure I'm following what you mean by type? here in this line we don't have the "httpMethod + endpoint" because it's a schema change, which is described in components

Reply:
ah, reread this and now i got it! yes that will be better

Reply:
update: I'm trying to update this but I seem to have found a bug in the unmarshalling/test. We are unmarshalling response as ApiChange, which leads to the wrong formatting. Checking if we can unmarshal w/ Change instead

Reply:
wont stress about trying to unmarshall different types as Changes is an interface, we can easilly just update this file w/ correct expectation (which is what we display)